src/app/(sidebar)/transaction/build/components/ClassicOperation.tsx

```diff
@@ -1,4 +1,4 @@
-import { ChangeEvent, Fragment, useState } from "react";
+import React, { ChangeEvent, Fragment, useState } from "react";
```

Comment:
do we need this?

Reply:
My IDE automatically added it because it prefers having it imported instead of using global. But we don't need it there, so I'll remove it.